ci/scripts/generate-dns-values.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash
+
+set -e
+
+test $DNS_VAR_FILE
+
+PROMETHEUS_EXTERNAL_HOSTNAME=$(dig +short public.prometheus.monitoring.cld.internal ptr)
+PROMETHEUS_EXTERNAL_HOSTNAME=${PROMETHEUS_EXTERNAL_HOSTNAME:0:-1} # Drop trailing period
+PROMETHEUS_EXTERNAL_IP=$(dig +short ${PROMETHEUS_EXTERNAL_HOSTNAME})
+
+ALERTMANAGER_EXTERNAL_HOSTNAME=$(dig +short public.alertmanager.monitoring.cld.internal ptr)
+ALERTMANAGER_EXTERNAL_HOSTNAME=${ALERTMANAGER_EXTERNAL_HOSTNAME:0:-1} # Drop trailing period
+ALERTMANAGER_EXTERNAL_IP=$(dig +short ${ALERTMANAGER_EXTERNAL_HOSTNAME})
+
+GRAFANA_EXTERNAL_HOSTNAME=$(dig +short public.grafana.monitoring.cld.internal ptr)
+GRAFANA_EXTERNAL_HOSTNAME=${GRAFANA_EXTERNAL_HOSTNAME:0:-1} # Drop trailing period
+GRAFANA_EXTERNAL_IP=$(dig +short ${GRAFANA_EXTERNAL_HOSTNAME})
+
+
+cat > $DNS_VAR_FILE <<END_OF_OPS
```

Comment:
Since the next line writes this out again, maybe just:

```bash
cat <<END_OF_OPS
...
END_OF_OPS
```

would be simpler?

Reply:
Hmm, I think I can see how this is used now. Would it be simpler if the script always just generated the data, printed to stdout, and then the task in the pipeline wrote it out with `> xxxx`?

That way the task owns all the logic, and the script just does what the filename says it does.

Reply:
Looking further at how a task is defined in Concourse, that's probably too hard to be practical, so feel free to disregard.